aries-site/src/pages/components/data.mdx

```diff
@@ -12,10 +12,10 @@ import {
   ToolbarExample,
 } from '../../examples/components/';
 
-Data is the wrapper component and orchestrator for a set of subcomponents. Because this set of components work together to create the entirety of the experience, we like to refer to them as "Data and friends".
+Because this set of components work together to create the entirety of the experience, we like to refer to them as "Data and friends".
```

Comment:
I feel like Im asking myself "what set of components" while reading this

Reply:
Should I maybe keep the first sentence, but just say "Data is the orchestrator for a set of subcomponents." (just cut out the "wrapper" bit)

Reply:
^ I agree the beginning also threw me off, felt like it needed a little more of an intro

Reply:
Updated this